--- origsrc/dialog-1.1-20080316/makefile.in	2007-07-04 14:33:35.000000000 -0500
+++ src/dialog-1.1-20080316/makefile.in	2008-03-23 15:34:08.812500000 -0500
@@ -126,7 +126,7 @@
 	$(RANLIB) $@
 
 dialog$x : $(LIB) dialog$o @INTLDIR_MAKE@ @INTLLIBS@
-	$(LINK) -o $@ dialog$o -L. -ldialog $(LDFLAGS) $(LIBS)
+	$(LINK) -o $@ dialog$o $(LIB) $(LDFLAGS) $(LIBS)
 
 clean	\
 distclean \
@@ -188,10 +188,10 @@
 	@ echo "** installing library in $(LIBDIR)"
 	@ $(LIBTOOL_INSTALL) $(INSTALL_DATA) $(LIB) $(LIBDIR)
 	@ echo "** installing headers in $(INCLUDEDIR)"
-	@ $(SHELL) $(srcdir)/headers-sh $(INSTALL_DATA) $(INCLUDEDIR) $(srcdir) dialog.h
-	@ $(SHELL) $(srcdir)/headers-sh $(INSTALL_DATA) $(INCLUDEDIR) $(srcdir) dlg_colors.h
-	@ $(SHELL) $(srcdir)/headers-sh $(INSTALL_DATA) $(INCLUDEDIR) $(srcdir) dlg_keys.h
-	@ $(SHELL) $(srcdir)/headers-sh $(INSTALL_DATA) $(INCLUDEDIR) .         dlg_config.h
+	@ $(SHELL) headers-sh $(INSTALL_DATA) $(INCLUDEDIR) $(srcdir) dialog.h
+	@ $(SHELL) headers-sh $(INSTALL_DATA) $(INCLUDEDIR) $(srcdir) dlg_colors.h
+	@ $(SHELL) headers-sh $(INSTALL_DATA) $(INCLUDEDIR) $(srcdir) dlg_keys.h
+	@ $(SHELL) headers-sh $(INSTALL_DATA) $(INCLUDEDIR) .         dlg_config.h
 
 uninstall-lib :: $(BINDIR)
 	$(RM) $(BINDIR)/dialog-config
@@ -209,8 +209,8 @@
 uninstall-lib ::
 	$(RM) $(MAN3DIR)/$(PACKAGE).3
 
-headers.sed : $(srcdir)/headers-sh
-	$(SHELL) $(srcdir)/headers-sh $(INCLUDEDIR) $(srcdir)
+headers.sed : headers-sh
+	$(SHELL) headers-sh $(INCLUDEDIR) $(srcdir)
 
 $(MAN1DIR) \
 $(MAN3DIR) \
